arm/Microsoft.Automation/automationAccounts/deploy.bicep

```diff
@@ -306,8 +334,11 @@ module automationAccount_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment,
 @description('The name of the deployed automation account')
 output automationAccountName string = automationAccount.name
 
-@description('The id of the deployed automation account')
+@description('The ID of the deployed automation account')
 output automationAccountResourceId string = automationAccount.id
 
 @description('The resource group of the deployed automation account')
 output automationAccountResourceGroup string = resourceGroup().name
+
+@description('The resource ID of the assigned identity.')
+output assignedIdentityID string = systemAssignedIdentity ? automationAccount.identity.principalId : ''
```

Comment:
`assignedIdentityResourceId`

Reply:
PrincipalID in that case. Its not the /subscription/***/ is of it, as far as i know. Its the GUID principal ID. Agree to call it `assignedIdentityPrincipalID`?

Reply:
Aligned with the api output, using `principalID` as output

Reply:
Agreed